annotator/static/views/player.js

```diff
@@ -46,6 +46,12 @@ class PlayerView {
         // Video URL
         this.videoSrc = videoSrc;
 
+        // Video start time
+        this.videoStart = parseFloat(videoStart);
```

Comment:
What do you mean to parse it in the backend? As in when I get it in views.py, use JSON.parse on the two variables?